modules/images/webp-uploads/load.php

```diff
@@ -8,6 +8,39 @@
  * @package performance-lab
  */
 
+/**
+ * Can load function to determine if the module is already merged into WordPress core or loaded by
+ * another plugin.
+ *
+ * @since 1.0.0
+ *
+ * @return boolean
+ */
+function webp_uploads_can_load() {
+	$can_load = require __DIR__ . '/can-load.php';
+	if ( $can_load() ) {
+		return true;
+	}
+	add_action(
+		'admin_notices',
+		function() {
+			printf(
+				'<div class="notice notice-error"><p>%s</p></div>',
+				sprintf(
+					__( 'The module is already merged into WordPress core or loaded by another plugin.', 'performance-lab' )
```

Comment:
See above. Also please clarify which module, plus we shouldn't use the word "module" since this message would primarily be for the plugin.
```suggestion
					__( 'The WebP Uploads feature cannot be loaded from within the plugin since it is already merged into WordPress core.', 'performance-lab' )
```